schema/reimbursement_main.sql

```diff
@@ -0,0 +1,71 @@
+-- phpMyAdmin SQL Dump
+-- version 4.8.3
+-- https://www.phpmyadmin.net/
+--
+-- Server version: 5.7.24-log
+-- PHP Version: 7.2.10
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `tribunal`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `reimbursement_main`
+--
+
+CREATE TABLE `reimbursement_main` (
+  `uid` int(11) NOT NULL,
+  `name` text NOT NULL,
+  `position` text NOT NULL,
+  `email` text NOT NULL,
+  `mid` text NOT NULL,
+  `date` date NOT NULL,
+  `vendor` text NOT NULL,
+  `amount` int(11) NOT NULL,
+  `description` text NOT NULL,
+  `status` text NOT NULL,
+  `type` text NOT NULL,
+  `receipt_name` text,
+  `document_name` text,
```

Comment:
Use `varchar` instead of `text` as you're already specifying an upper limit to this field in the PHP and it's a little bit more performant.